src/BookSelling.sol

```diff
@@ -35,14 +35,13 @@ contract BookSelling is Ownable{
     }
 
     struct Order {
-        uint256[] bookIds;
-        uint256[] prices;
-        uint256[] quantities;
+        uint256 orderId;
+        Book[] books;
```

Comment:
Regroup types in Solidity ;)

Reply:
Should be 
```
uint
uint
Book
Customer
```

Reply:
Yep, removed that struct anyways as not used, but did so in the event as well, thanks